boltdesignsystem/bolt
# DS-798 Grid object deprecation

## Jira

https://pegadigitalit.atlassian.net/browse/DS-798

## Summary

Grid object is deprecated.

## Details

The grid object was replaced with `bolt-grid-component`.

## How to test

Pull the branch. Check if all examples of the grid-object were adequately replaced with the grid-component in the twig files.

## Release notes

Grid object is deprecated. Please use the [Grid component](https://boltdesignsystem.com/pattern-lab/?p=viewall-components-grid) instead.
